wrapanapi/systems/ec2.py

```diff
@@ -531,7 +553,7 @@ def id(self) -> str:
         This part is used as id in aws cli.
         """
         if self.arn:
-            return self.arn.split(":")[-1]
+            return self.arn.split(":")[-1].split("/")[-1]
```

Comment:
looking at the docs here https://docs.aws.amazon.com/quicksight/latest/APIReference/qs-arn-format.html it could be three formats 

```
arn:partition:service:region:account-id:resource-id
arn:partition:service:region:account-id:resource-type/resource-id
arn:partition:service:region:account-id:resource-type:resource-id
```
it will fail for others ? I think you need to handle that ?

Reply:
Resolved in [latest commit](https://github.com/RedHatQE/wrapanapi/pull/481/commits/ade36a5d8416c1dccf4477ca4ca429ef4f13e64b#diff-733f95452f0413255428ec1bbd8485f7a880c27d54942cb97c72a9f40be328eaR553-R555)